SPECS/node-problem-detector/node-problem-detector.spec

```
@@ -1,13 +1,14 @@
 Summary:        Kubernetes daemon to detect and report node issues
 Name:           node-problem-detector
-Version:        0.8.8
+Version:        0.8.10
 Release:        1%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Daemons
 URL:            https://github.com/kubernetes/node-problem-detector
-Source0:        https://github.com/kubernetes/%{name}/archive/refs/tags/%{name}-%{version}.tar.gz
+#Source0:        https://github.com/kubernetes/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```

Please use this instead (makes spec parsing easier):
```suggestion
Source0:        https://github.com/kubernetes/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```